.travis.yml

```diff
@@ -38,7 +38,6 @@ matrix:
       name: SwiftPM / Linux / Swift 4.2.4
       os: linux
       sudo: required
-      dist: trusty
```

Comment:
The current default dist is Xenial (16.04) now: https://docs.travis-ci.com/user/reference/overview/#linux

![image](https://user-images.githubusercontent.com/909674/74304042-30145880-4d9f-11ea-8492-99869856e5ff.png)